website/docs/d/policy_set_definition.html.markdown

```diff
@@ -42,10 +42,26 @@ output "id" {
 
 * `policy_definitions` - The policy definitions contained within the policy set definition.
 
+* `policy_definition_reference` - One or more `policy_definition_reference` blocks as defined below.
+
 * `parameters` - Any Parameters defined in the Policy Set Definition.
 
 * `metadata` - Any Metadata defined in the Policy Set Definition.
 
+---
+
+An `policy_definition_reference` block exports the following:
+
+* `policy_definition_id` - The ID of the policy definition or policy set definition that is included in this policy set definition.
+
+* `parameters` - The mapping of the parameter values for the referenced policy rule. The keys are the parameter names.
+
+-> **NOTE:** Since Terraform's concept of a map requires all of the elements to be of the same type, the value in parameters will all be converted to string type.
```

Comment:
```suggestion
-> **NOTE:** Since Terraform's concept of a map requires all of the elements to be of the same type, the value in parameters will all be converted to string type.

~> **Note:** This field only supports String fields and is deprecated in favour of the `parameters_values` field
```

Reply:
Done